app/assets/stylesheets/base/_logos.scss

```diff
@@ -1,22 +1,31 @@
-$logo-width-mobile: 96px;
-$logo-width-tablet: 135px;
-$logo-width-desktop: 219px;
+$logo-wcmc-width-mobile: 96px;
+$logo-wcmc-width-tablet: 135px;
+$logo-wcmc-width-desktop: 219px;
 
 $logo-height-mobile: 22px;
 $logo-height-tablet: 31px;
 $logo-height-desktop: 50px;
 
+$logo-unep-width-mobile: 96px;
+$logo-unep-width-tablet: 135px;
+$logo-unep-width-desktop: 219px;
```

Comment:
As those variable have the exact same value, there is probably no point having 2 sets (wcmc and unep). Maybe we can keep the original version ?